clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -122,9 +122,9 @@
                                         Importance.LOW,
                                         RECONNECT_BACKOFF_MAX_MS_DOC)
                                 .define(RETRY_BACKOFF_MS_CONFIG,
-                                        Type.LONG,
-                                        100L,
-                                        atLeast(0L),
+                                        Type.INT,
```

Comment:
We may want to keep the type of retryBackoffMs of AdminClient to be the same as that of the ProducerConfig and ConsumerConfig. Is there specific benefits of changing it to int?

Reply:
We can leave it consistent for now. Just thought it was unlikely anyone would want to backoff for longer than 24 days 😉 .

Reply:
Yeah it is totally reasonable to use int for retryBackoff for this reason. It is just a bit weird for it to be inconsistent with ProducerConfig and ConsumerConfig. These are all configuration exposed to user. If we decide `int` it better, maybe we should change retryBackoffMs to be int for all three classes for the same reason?

Reply:
Yeah, let's do it separately I guess. There are a few more of these that also don't make sense as long. For example, the reconnect backoffs.